catalog/ui/catalog-ui-search/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -182,11 +182,57 @@
         <argument ref="catalogFramework"/>
     </bean>
 
+    <bean id="querySortsParser"
+          class="org.codice.ddf.catalog.ui.metacard.workspace.transformations.QuerySortsParser"/>
+
+    <service ref="querySortsParser"
+             interface="org.codice.ddf.catalog.ui.metacard.workspace.transformer.WorkspaceTransformation"/>
+
+    <bean id="workspaceMetacardsKeyTransformation"
+          class="org.codice.ddf.catalog.ui.metacard.workspace.transformations.WorkspaceMetacardsKeyToAssociations"/>
+
+    <service ref="workspaceMetacardsKeyTransformation"
+             interface="org.codice.ddf.catalog.ui.metacard.workspace.transformer.WorkspaceTransformation"/>
+
+    <bean id="srcKeyTransformation"
+          class="org.codice.ddf.catalog.ui.metacard.workspace.transformations.SrcToQuerySources"/>
+
+    <service ref="srcKeyTransformation"
+             interface="org.codice.ddf.catalog.ui.metacard.workspace.transformer.WorkspaceTransformation"/>
+
+    <bean id="removeMetacardTagsTransformation"
+          class="org.codice.ddf.catalog.ui.metacard.workspace.transformations.RemoveMetacardTags"/>
+
+    <service ref="removeMetacardTagsTransformation"
+             interface="org.codice.ddf.catalog.ui.metacard.workspace.transformer.WorkspaceTransformation"/>
+
+    <bean id="queryMetacardsTransformation"
+          class="org.codice.ddf.catalog.ui.metacard.workspace.transformations.EmbeddedMetacardsHandler" factory-method="newQueryMetacardHandler"/>
+
+    <service ref="queryMetacardsTransformation"
+             interface="org.codice.ddf.catalog.ui.metacard.workspace.transformer.WorkspaceTransformation"/>
+
+    <bean id="listMetacardsTransformation"
+          class="org.codice.ddf.catalog.ui.metacard.workspace.transformations.EmbeddedMetacardsHandler" factory-method="newListMetacardHandler"/>
+
+    <service ref="listMetacardsTransformation"
+             interface="org.codice.ddf.catalog.ui.metacard.workspace.transformer.WorkspaceTransformation"/>
+
+    <bean id="removeBlankAccessIndividualsTransformation"
+          class="org.codice.ddf.catalog.ui.metacard.workspace.transformations.FilterBlankAccessIndividuals"/>
+
+    <service ref="removeBlankAccessIndividualsTransformation"
```

Comment:
why do we publish these?  do they actually get used elsewhere?

Reply:
They are published to match the `reference-list` argument in the `workspaceTransformer` below. They are declared as `WorkspaceTransformation`s matched by a `reference-list` instead of simply being passed in as a hardcoded list argument in order to allow downstream projects to extend the functionality of the `WorkspaceTransformer` by declaring more `WorkspaceTransformation`s.